### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by wawachaan** [ikhsan3adi/absensi-sekolah-qr-code#77] [General]
halo bang boleh nanya gk

**Comment by ikhsan3adi** [comment]
Ya boleh

**Comment by MURIFA** [comment]
Mau tau dong cara instalasi dari awal sampe bisa buka website nya

**Reply by ikhsan3adi** [reply]
Ada di readme :)

**Comment by antoseno** [comment]
gan, mau tanya kalau untuk rekap hasil absensinya di buat menjadi 6 hari kerja ( sampai hari sabtu) bagaimana ya?

**Reply by wayangm** [reply]
Up butuh juga, adakah yang bisa bantu jawab..

**Comment by Eri31-lgtm** [comment]
Mas mau tanya
Saya sudah menggunakan aplikasi di sekolah dengan versi sebelumnya, dan tidak mengalami update sama sekali. 

Apakah kalo saya download yang versi terbaru ini v.1.7.0 akan berjalan normal ? 
Terimakasih sebelumnya

**Reply by mdestafadilah** [reply]
Backup dulu datanya, ada banyak fiture yang akan segera release.
